ember-cli-build.js

```diff
@@ -1,8 +1,8 @@
 /* eslint-env node */
-var EmberAddon = require('ember-cli/lib/broccoli/ember-addon')
+const EmberAddon = require('ember-cli/lib/broccoli/ember-addon')
 
 module.exports = function (defaults) {
-  var app = new EmberAddon(defaults, {
+  const app = new EmberAddon(defaults, {
```

Comment:
This change is needed for the already merged lint rules in `eslint-config-frost-standard` is met. I thought that by putting that rule there, we'd already agreed we wanted all repos to no longer use `var`. There was an issue I was unaware of, where this repo needs to have a more permissive `dependencies` set so that a consumer can actually use the already published `eslint-config-frost-standard`. Currently, if the user explicitly installs `eslint-config-frost-standard@7.0.0`, and also has this addon, then this addon will use it's own `eslint-config-frost-standard@6.x` and so the consumer gets out-of-date rules. As part of allowing this addon to work with the latest `eslint-config-frost-standard`, I need to fix the lint errors within this repo, or add ignores everywhere it differs. Personally, I'd rather have the addon follow our established lint rules than have it match what `ember-cli` spits out by default.

I don't particularly care about the styling of this addon, or of any other addons. I simply want to be able to use the latest `eslint-config-frost-standard` in my app. As long as I'm using this addon, I can't do that without these changes. Do you have a better solution for how to allow apps to actually use the latest lint standards?